### DISCUSSION TITLE
Homeproxy 自定义模式懒人配置脚本。有需要请自取

### DISCUSSION THREAD

**Discussion by thisIsIan-W** [immortalwrt/homeproxy#258] [Show and tell]
仓库里代码已经过大量测试，目前基本稳定无bug。
有需要配置大量规则集的朋友可以尝试使用此脚本来解放双手  ---->  [仓库链接](https://github.com/thisIsIan-W/homeproxy-autogen-configuration/tree/main)

**Comment by thisIsIan-W** [comment]
具体的使用说明、注意事项等在仓库中有提供，请仔细阅读。
如遇到脚本bug，请直接在仓库提issue！

**Comment by ATang007ZH** [comment]
不够懒人啊。懒人是只需要点几个按钮就ok的。

**Reply by thisIsIan-W** [reply]
这么大人了还得让别人一口口给你喂嘴里么？
理解能力实在有问题的话就别出来现眼了~~Peace ; )

**Reply by muink** [reply]
一次配置长久使用也是可以偷懒了
而且玩自定义模式多少还是要动点脑子的

**Reply by ATang007ZH** [reply]
> 这么大人了还得让别人一口口给你喂嘴里么？ 理解能力实在有问题的话就别出来现眼了~~Peace ; )

为啥说 现眼 这么文明的字眼，显的你修养高吗。
每个人对懒人的定义不同，有异议很正常啊。

**Reply by ATang007ZH** [reply]
> 一次配置长久使用也是可以偷懒了 而且玩自定义模式多少还是要动点脑子的

这么说的话这个确实可以偷懒的了。

对我们刚接触hp的小白来说，想玩自定义模式确实有点懵，主要是里面全部是空白的，一点可以参考的东西都没有，也没有wiki文档可以学习。

**Reply by thisIsIan-W** [reply]
> > 这么大人了还得让别人一口口给你喂嘴里么？ 理解能力实在有问题的话就别出来现眼了~~Peace ; )
> 
> 为啥说 现眼 这么文明的字眼，显的你修养高吗。 每个人对懒人的定义不同，有异议很正常啊。

别想太复杂，单纯就是因为像你这样的喷子不配~~~  ; )

首先，您要是真有需求，欢迎把你具体的、可行的思路发布到issue里，我看心情决定要不要更新~~~ ; )

还有，真别动不动就说自己是小白了，太侮辱这个词儿了。
脚本从第一版发布至今就是靠很多真正的小白一点点贡献出自己的想法和真诚的建议才逐渐完善成为今天的样子，包括 homeproxy 插件本身也是这样。相比于那些真正的小白用户来说，称呼您一句小丑我私心觉得可太合适了。

最后，如果你真的太缺乏动手解决问题的能力而只会抱怨和人云亦云，真诚建议您从您的所有设备上删除脚本和 HP 插件。

------------------------------------------------------------------------------------------------------------------------------------

毕竟如果你要把你家电费也赖开源作者头上，我们也没地儿伸冤不是？

Peace ; )

**Reply by ATang007ZH** [reply]
> > > 这么大人了还得让别人一口口给你喂嘴里么？ 理解能力实在有问题的话就别出来现眼了~~Peace ; )
> > 
> > 
> > 为啥说 现眼 这么文明的字眼，显的你修养高吗。 每个人对懒人的定义不同，有异议很正常啊。
> 
> 别想太复杂，单纯就是因为像你这样的喷子不配~~~ ; )
> 
> 首先，您要是真有需求，欢迎把你具体的、可行的思路发布到issue里，我看心情决定要不要更新~~~ ; )
> 
> 还有，真别动不动就说自己是小白了，太侮辱这个词儿了。 脚本从第一版发布至今就是靠很多真正的小白一点点贡献出自己的想法和真诚的建议才逐渐完善成为今天的样子，包括 homeproxy 插件本身也是这样。相比于那些真正的小白用户来说，称呼您一句小丑我私心觉得可太合适了。
> 
> 最后，如果你真的太缺乏动手解决问题的能力而只会抱怨和人云亦云，真诚建议您从您的所有设备上删除脚本和 HP 插件。
> 
> 毕竟如果你要把你家电费也赖开源作者头上，我们也没地儿伸冤不是？
> 
> Peace ; )

真的很奇怪，
我只是调侃说了句玩笑话而已，
具体的思路应该是发布到讨论里的了，等您心情好的时候再说吧
咋小丑就不打搅您这心情了。

**Reply by thisIsIan-W** [reply]
> > > > 这么大人了还得让别人一口口给你喂嘴里么？ 理解能力实在有问题的话就别出来现眼了~~Peace ; )
> > > 
> > > 
> > > 为啥说 现眼 这么文明的字眼，显的你修养高吗。 每个人对懒人的定义不同，有异议很正常啊。
> > 
> > 
> > 别想太复杂，单纯就是因为像你这样的喷子不配~~~ ; )
> > 首先，您要是真有需求，欢迎把你具体的、可行的思路发布到issue里，我看心情决定要不要更新~~~ ; )
> > 还有，真别动不动就说自己是小白了，太侮辱这个词儿了。 脚本从第一版发布至今就是靠很多真正的小白一点点贡献出自己的想法和真诚的建议才逐渐完善成为今天的样子，包括 homeproxy 插件本身也是这样。相比于那些真正的小白用户来说，称呼您一句小丑我私心觉得可太合适了。
> > 最后，如果你真的太缺乏动手解决问题的能力而只会抱怨和人云亦云，真诚建议您从您的所有设备上删除脚本和 HP 插件。
> > 毕竟如果你要把你家电费也赖开源作者头上，我们也没地儿伸冤不是？
> > Peace ; )
> 
> 真的很奇怪， 我只是调侃说了句玩笑话而已， 具体的思路应该是发布到讨论里的了，等您心情好的时候再说吧 咋小丑就不打搅您这心情了。

确实，动不动就调侃别人挺奇怪的🫶🏼🫶🏼🫶🏼

**Reply by choicky** [reply]
多谢贴主，如没有这个脚本，我还真的没办法配置好自定义的模式

**Reply by Todd-018** [reply]
感谢贴主分享！

**Comment by ALin-buildNow** [comment]
"真"一键配置！支持贴主！

**Comment by grdws** [comment]
用你这个配置，遇到个现象一头雾水，维基百科 https://zh.wikipedia.org 打不开，我猜测是域名匹配不上规则集。为了单独测试这个网站，把所有路由规则的出站全选择成国外的节点，把DNS设置的默认DNS服务器选择成国外的服务器，把所有DNS规则的服务器全选择成国外的服务器，我觉得这样设置，维基百科应该用国外的DNS解析并且应该走代理节点，可这个网站还是打不开。假如维基百科的域名匹配不上，homeproxy的流程是怎么走的？
